incubator/hnc/e2e/tests/delete-anchor-crd_test.go

```diff
@@ -0,0 +1,35 @@
+package test
+
+import (
+	"os/exec"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Delete-anchor-crd", func() {
+	It("should create parent and deletable child", func() {
+		// create a namespace delete-crd-parent
+		cmd := exec.Command("kubectl", "create", "ns", "delete-crd-parent")
+		Expect(cmd.Run()).Should(BeNil())
+
+		// create a namespace delete-crd-child and set its parent as delete-crdparent
+		cmd = exec.Command("kubectl", "hns", "create", "delete-crd-child", "-n", "delete-crd-parent")
+		Expect(cmd.Run()).Should(BeNil())
+
+		// allow cascading delete for delete-crd-child
+		cmd = exec.Command("kubectl", "hns", "set", "delete-crd-child", "--allowCascadingDelete")
+		Expect(cmd.Run()).Should(BeNil())
+	})
+
+	It("should delete the CRD", func() {
+		// delete the crd
+		cmd := exec.Command("kubectl", "delete", "ns", "customresourcedefinition.apiextensions.k8s.io/subnamespaceanchors.hnc.x-k8s.io")
+		Expect(cmd.Run()).Should()
+		// TODO: @ginnyji - output should be "customresourcedefinition.apiextensions.k8s.io "subnamespaceanchors.hnc.x-k8s.io" deleted"
```

Comment:
Do you _need_ to check that text output? Or will it return a non-nil error if it fails?

Reply:
Or just have a expected failed `kubectl get crd subnamespaceanchor` after the deletion?

Reply:
The problem was because of the extra 'ns'. We should be expecting a Nil